install/kubernetes/helm/meshery-operator/charts/meshery-cilium/Chart.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v2
+name: meshery-cilium
+description: Meshery Adapter for Cilium chart.
+source: 
+  - https://github.com/meshery/meshery-cilium
+
+deprecated: false
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
```

Comment:
Same consideration here.

Reply:
I'm afraid not. The Action we used only support the primary chart(meshery-operator). If I remember right.